src/halo/k8s/cortx-functions.sh

```diff
@@ -0,0 +1,360 @@
+#!/bin/bash
```

Comment:
There should not be anything named cortx inside halo directory. Rename this file to k8s-function.sh or something similar.

Reply:
done